src/orange/orangeinp/StackedExtrudedPolygon.cc

```diff
@@ -155,10 +155,10 @@ NodeId StackedExtrudedPolygon::make_levels(
 
     // Create a vector of all concave regions, via recursion
     std::vector<NodeId> concave_nodes(concave_regions.size());
-    for (auto const i : range(concave_nodes.size()))
+    for (size_type i : range(concave_nodes.size()))
```

Comment:
Don't bother changing it, but the "best practices" (Scott Myers) is to static cast where needed and use auto rather than an implicit cast:
```suggestion
    for (auto i : range(static_cast<size_type>(concave_nodes.size())))
```

Reply:
Gotcha. I will change that way I can reference it in the future.